frontends/web/src/routes/account/utils.ts

```diff
@@ -15,7 +15,21 @@
  * limitations under the License.
  */
 
-import { CoinCode, ScriptType } from '../../api/account';
+import { CoinCode, ScriptType, IAccount } from '../../api/account';
+
+export function getAccount(accounts: IAccount[], accountCode: string | undefined): IAccount | undefined {
+  if (!accounts) {
+    return undefined;
+  }
+  return accounts.find(({ code }) => accountCode === code);
+}
```

Comment:
There is an api function with a similar name `getAccounts` in `api/account.ts`, please rename getAccount to something else, maybe findAccount ?

Also there is an  almost identical function in send.tsx see https://github.com/digitalbitbox/bitbox-wallet-app/blob/936332e17d9949836d44f572cfbaedaa3c5fda44/frontends/web/src/routes/account/send/send.tsx#L475-L480

Worth refactoring? Also you could move this whole file to `src/utils`.

Reply:
Function renamed and `send.tsx` refactored. I would move the file in a separate PR, since that involves changes to several other files.